docs/docs/contribute/getting-started.md

```diff
@@ -14,109 +14,193 @@ license: CC-BY-SA-4.0
+
+    !!! help
+        You can check if Rust is properly installed by using `cargo --version` in the Command Prompt. 
+
+        If a version number is printed, your installation was successful!
+            ```
+            cargo --version
+            ```
+
+    We will be using the nightly version of Rust which is updated more frequently compared to the stable and beta versions. Install it with the following commands:
+        ```cmd
+        rustup toolchain install nightly
+        rustup +nightly target add x86_64-pc-windows-msvc
+        rustup default nightly-x86_64-pc-windows-msvc
         ```
 
-We will be using the nightly version of Rust which is updated more frequently compared to the stable and beta versions. Install it with the following commands:
-    ```
-    rustup toolchain install nightly
-    ```
-    ```
-    rustup +nightly target add x86_64-pc-windows-gnu
-    ```
-	```
-    rustup default nightly-x86_64-pc-windows-gnu
-	```
+=== "Linux"
+
+    !!! help
+        You can check if Rust is properly installed by using `cargo --version` in the Command Prompt. 
+
+        If a version number is printed, your installation was successful!
+            ```
+            cargo --version
+            ```
+
+    We will be using the nightly version of Rust which is updated more frequently compared to the stable and beta versions. Install it with the following commands:
+        ```bash
+        rustup toolchain install nightly
+        rustup +nightly target add x86_64-unknown-linux-gnu
+        rustup default nightly-x86_64-unknown-linux-gnu
+        ```
 
 To add support for building to web browsers, input the following command:
-	```
+    ```
     rustup +nightly target add wasm32-unknown-unknown
-	```
+    ```
 
 Use the `rustup update` command to update your installation. Since we are using the nightly release, you are encouraged to update often.
-	```
+    ```
     rustup update
-	```
- 
-<!-- For Linux / WSL
+    ```
 
-Only 2 commands are required to set up the Rust nightly build:
 
-rustup install nightly
-rustup default nightly
+##  Git & GitHub
 
--->
+Git is a system used for managing and tracking changes in your code. It is also known as a Version Control System (VCS). It makes it easier for collaborators to work together, and allows you to access our projects too!
 
-##  Git & GitHub ##
+=== "Windows"
+    Install Git [here](https://git-scm.com/download). The installation settings can be left unchanged.
 
-Git is a system used for managing and tracking changes in your code. It is also known as a Version Control System (VCS). It makes it easier for collaborators to work together, and allows you to access our projects too!
+    Alternatively, you can also use chocolatey:
+    ```cmd
+    choco install -y git
+    ```
 
-Install Git [here](https://git-scm.com/download). The installation settings can be left unchanged.
+=== "Linux"
+    ```bash
```

Comment:
You might want an opening line to explain what the command does.

For example:
`Install Git with the following command:`